tests/pytests/unit/modules/test_event.py

```diff
@@ -0,0 +1,107 @@
+"""
+    :codeauthor: Rupesh Tare <rupesht@saltstack.com>
+
+    Test cases for salt.modules.event
+"""
+
+import pytest
+
+import salt.modules.event as event
+import salt.utils.event
+from tests.support.mock import MagicMock, patch
+from tests.support.paths import SALT_CODE_DIR
+
+
+@pytest.fixture
+def minion_opts(minion_opts):
+    minion_opts.update(
+        {
+            "id": "id",
+            "sock_dir": SALT_CODE_DIR,
```

Comment:
```suggestion
```
This is not necessary, and probably transport too.

Reply:
Actually, looking at the tests, this whole fixture shouldn't be necessary.

Reply:
Removed `sock_dir` and `transport`. 

`id` seems to be needed for me to get the tests to pass.

Unless I am doing something wrong, which is highly possible. 

Thanks for input.